operations/governance-remediation-process.md

```diff
@@ -0,0 +1,83 @@
+# Governance Remediation Escalation 
+
+This governance remediation process is for cases where CNCF projects are either not practicing their governance as documented or the project is stuck in a circumstance that its current governance does not cover. As projects evolve over time, we do expect their governance to evolve as well, the key being the documentation gets updated and reflects current practice(s) quickly before there is a crisis. For projects that are missing parts of their governance and would like help adding them, please consider opening a [governance review issue](https://github.com/cncf/tag-contributor-strategy/issues/new?assignees=jberkus%2Caliok%2Cgeekygirldawn&labels=wg%2Fgovernance&projects=&template=governance-review-request.yaml&title=%5BGovernance+Review%5D%3A+PROJECT+NAME) with TAG Contributor Strategy.
```

Comment:
```suggestion
This governance remediation process is for cases where CNCF projects are either not practicing their governance as documented or the project is stuck in a circumstance that its current governance does not cover. As projects evolve over time, we expect projects to evolve their governance as well, the key being the documentation gets updated and reflects current practice(s) quickly before there is a crisis. For projects that would like help adding to or updating their governance, please consider opening a [governance review issue](https://github.com/cncf/tag-contributor-strategy/issues/new?assignees=jberkus%2Caliok%2Cgeekygirldawn&labels=wg%2Fgovernance&projects=&template=governance-review-request.yaml&title=%5BGovernance+Review%5D%3A+PROJECT+NAME) with TAG Contributor Strategy.
```

Minor wording suggestion, intending that the normal path is for projects to update their own governance independently; also that the issue might not always be "missing" governance, it could be out-dated